pages/administratorDashboard/administratorDashboard.ejs

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<%- include('../partials/head'); %>
+<link href="/stylesheets/theme.bootstrap.css" rel="stylesheet" />
+<script src="/javascripts/jquery.tablesorter.min.js"></script>
+<script src="/javascripts/jquery.tablesorter.widgets.js"></script>
+</head>
+
+<body>
+<%- include('../partials/navbar', {navPage: 'adminDashboard'}); %>
+
+<%
+function linkify(row, col) {
+
+  var href= null;
+
+  if (col == 'course_id') {
+    href = urlPrefix + '/course/' + row[col];
+  } else if (col == 'course_instance_id') {
+    href = urlPrefix + '/course_instance/' + row[col];
+  } else if (col == 'assessment_id' && 'course_instance_id' in row) {
+    href = urlPrefix + '/course_instance/' + row['course_instance_id'] + '/instructor/assessment/' + row[col];
+  }
+
+  if (href) { %>
+<a href="<%= href %>"><%= row[col] %></a>
+  <% } else { %>
+<%= row[col] %>
+  <% } 
+}; %>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-4 bg-light">
+            <h1>SQL Dashboards</h1>
+            <ul>
+                <% otherDashboards.forEach(function(dash) { %>
+                <li><a href="<%= baseUrl %>/<%= dash %>"><%= dash %></a></li>
+                <% }); %>
+            </ul>
+
+        </div>
+        <div class="col-md-4 bg-light">
+            <h2><%= dashboardName %></h2>
+            <ul>
+                <% queries.forEach(function(q) { %>
+                <li><a href="#<%= q.SQLFilename%>"><%= q.SQLFilename %></a>
+                <span class="badge badge-pill badge-info"><%= q.rowCount %></span></li>
+                <% }); %>
+            </ul>
+
+            As of <%= req_date_present %> UTC.
+        </div>
+    </div>
+    <div class="row">
+    <div class="col-12">
+
+<% queries.forEach(function(q) { 
+   var columns = [];
+%>
+
+<h2 id="<%= q.SQLFilename %>"><%= q.SQLFilename %>
+    <span class="badge badge-pill badge-info"><%= q.rowCount %></span></h2>
+<span class="badge badge-secondary"
+    title="<%= q.SQLstring %>">SQL</span>
```

Comment:
I had to read the EJS source to figure out how to use this button. Just make it a collapse element in the main page?